src/renderer/pages/project/script_editor/beat_editor.vue

```
@@ -338,22 +340,17 @@ const changeBeat = (beat: MulmoBeat) => {
 
 const generateImageOnlyImage = () => {
   const imageAgentInfo = MulmoPresentationStyleMethods.getImageAgentInfo(props.mulmoScript, props.beat);
-  if (!globalStore?.hasApiKey(imageAgentInfo.keyName)) {
-    notifyError("Error", "You need setup " + imageAgentInfo.keyName, {
-      label: "Setup",
-      onClick: () => globalStore.toggleSettingModal(),
-    });
+  if (!globalStore?.hasApiKey(imageAgentInfo.keyName) || true) {
```

ESLint

[error] Unexpected constant condition.

(no-constant-condition)